src/CameraParams.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "helper_math.h"
+
+struct CameraParams {
+    CameraParams(float3 lookFrom, float3 front, float vfov = 45.0f, float hfov = 45.0f) : 
+        front{front},
+        lookFrom{lookFrom},
+        vfov{vfov},
+        hfov{hfov} {}
+    float3 front;
+    float3 lookFrom;
+    float vfov = 45.0f;
+    float hfov = 45.0f;
+};
+
```

Comment:
NIT: I know that `struct` acts almost similar like class but maybe to avoid confusion we could just define it as `class` here?

Reply:
I prefer ```struct``` for objects without methods